go/vt/vttablet/tabletserver/schema/engine.go

```diff
@@ -126,15 +131,62 @@ func (se *Engine) InitDBConfig(cp dbconfigs.Connector) {
 	se.cp = cp
 }
 
+// syncVTDatabase is called either the first time a primary starts or on consequent loads to possibly upgrade to a
+// new Vitess version. This is the only entry point into the sidecardb module to get the _vt database to the desired
+// schema for the running Vitess version.
+// There is some extra logging in here which can be removed in a future version (>v16) once the new schema init
+// functionality is stable.
+func (se *Engine) syncVTDatabase(ctx context.Context, conn *dbconnpool.DBConnection) error {
+	log.Infof("In syncVTDatabase")
+	defer func(start time.Time) {
+		log.Infof("syncVTDatabase took %d ms", time.Since(start).Milliseconds())
+	}(time.Now())
+
+	var exec sidecardb.Exec = func(ctx context.Context, query string, maxRows int, wantFields bool, useVT bool) (*sqltypes.Result, error) {
```

Comment:
`useDB` is definitely better here IMO as it's clearer what it is and not tied to an underlying name.

Reply:
exec is always called with `wantFields` false. do we even need that parameter?
and the only place it is called with `useVT`=true is in a func that says
```
// createOrUpgradeTable expects that we are already in the _vt database
```
so I'm not sure why we need either of these two params to the func.

Reply:
`schemadiff` drops the db name when generating the alter, hence we need the `use _vt`.   I am going to see if we can add that as a `hint` in schemadiff and then drop it. I removed `wantFields`. Removed that comment